crates/forge/tests/integration/trace.rs

```diff
@@ -476,7 +476,6 @@ fn trace_library_call_from_test() {
 }
 
 #[test]
-#[ignore]
```

Comment:
If someone didn't review #1236 make sure to check the content of this test to see if everything behaves as intended (the test passes now)